internal/statemachine/classic_states.go

```diff
@@ -92,3 +92,20 @@ func (stateMachine *StateMachine) runLiveBuild() error {
 
 	return nil
 }
+
+// Generate the manifest
+func (stateMachine *StateMachine) generatePackageManifest() error {
+	// This is basically just a wrapper around dpkg-query
+
+	outputPath := filepath.Join(stateMachine.commonFlags.OutputDir, "filesystem.manifest")
+	cmd := execCommand("sudo", "chroot", stateMachine.tempDirs.rootfs, "dpkg-query", "-W", "--showformat=${Package} ${Version}\n")
+	manifest, err := os.Create(outputPath)
+	if err != nil {
+		return fmt.Errorf("Error creating manifest file: %s", err.Error())
```

Comment:
This line is not covered by existing tests. Mocking `os.Create` similar to how other `os.` calls are mocked would address that.